bookbotproject/telegrambot.py

```diff
@@ -80,24 +80,43 @@ def send_message(message, chat_id):
 
 
 def main():
-	counter = 0;
-	last_chat = (None, None);
-	last_update = None;
+	counter = 0
+	last_chat = (None, None)
+	last_update = None
+	end_switch = False
 	while True:
-		text, chat, update_id = get_last_chat(get_updates(last_update));
+
+		text, chat, update_id, user_id = get_last_chat(get_updates(last_update))
+		#print(text)
 		if(text, chat) != last_chat:
+
 			r = requests.get(HOST + "getResponse/{}/{}/".format(text, chat))
-			print(counter)
-			if r.text == "\"Which genre's books are you looking for?\"" and counter >0 :
-				print("here2")
+			#print(counter)
+			if end_switch and r.text == '\"Goodbye bookworm!\"':
+				end_switch = True
+			elif r.text == "\"Which genre's books are you looking for?\"" and counter >0 :
+				end_switch = False
+				#print("here2")
 				res = get_next_message_by_response(text, chat)
 				send_message(res, chat);
 				counter = 0
+			elif r.text == '\"Goodbye bookworm!\"':
+				send_message(r.text, chat);
+				counter = 0
+				end_switch = True
+			elif r.text == '\"Which book do you want to comment on?\"':
+				comment = text
+				send_message(r.text, chat);
+			elif r.text == '\"Your comment is saved!\"':
+				book = text
+				url =  HOST + "addComment/{}/{}/{}/".format(book, user_id, comment)
```

Comment:
neat. thank you.

Reply:
I've deleted them Irmak. thank you for the review.